### PR TITLE
ocamlPackages.labltk: init at 8.06

### PR DESCRIPTION
**Pull request by vbgl**
LablTk is an OCaml interface to the Tcl/Tk GUI framework.

homepage: http://labltk.forge.ocamlcore.org/

###### Motivation for this change

https://github.com/NixOS/nixpkgs/issues/43512#issuecomment-405051777
cc/ @ghasshee

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by FRidh** [comment]
@GrahamcOfBorg eval

File: pkgs/development/ocaml-modules/labltk/default.nix [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  buildFlags = [ "all" "opt" ];
+
+  createFindlibDestdir = true;
+
+  postInstall = ''
+    mkdir -p $OCAMLFIND_DESTDIR/stublibs
+    mv $OCAMLFIND_DESTDIR/labltk/dlllabltk.so $OCAMLFIND_DESTDIR/stublibs/
+  '';
+
+  meta = {
```
Comment (pSub) [inline pkgs/development/ocaml-modules/labltk/default.nix]: Mind adding yourself as a maintainer=?

**Review by pSub — COMMENTED** [review]
Looks fine otherwise.

File: pkgs/development/ocaml-modules/labltk/default.nix (hunk 1 above)
Comment (vbgl) [reply pkgs/development/ocaml-modules/labltk/default.nix]: done

**Pull request merged**